<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cadastro</class>
 <widget class="QMainWindow" name="Cadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Produtos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(193, 255, 230);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbTitulo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font:&quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Cadastro de Produtos</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>491</width>
      <height>266</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="LayoutProdutos">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="lbCodigo">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Codigo:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="inputCodigo">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbNome">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nome:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="inputNome">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbPreco">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Preço:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="inputPreco">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbQuantidade">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Quantidade:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="inputQuantidade">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lbDescricao">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Descrição:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="inputDescricao">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbCategoria">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Categoria:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbCategoria">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Informatica</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bebidas</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Congelados</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Utilidades</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Frutas</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnCadastrar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>500</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(175, 255, 137);</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbResposta">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>450</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(4, 4, 4);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 85);
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
selection-color: rgb(123, 255, 138);
selection-background-color: rgb(12, 12, 12);</string>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionTelaPrincipal"/>
   </widget>
   <addaction name="menuHome"/>
  </widget>
  <action name="actionCadastrar">
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </action>
  <action name="actionVisualizar">
   <property name="text">
    <string>Visualizar</string>
   </property>
  </action>
  <action name="actionTelaPrincipal">
   <property name="text">
    <string>Tela Inicial</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
